hot-desk/src/main/java/com/exadel/sandbox/parking_spot/repository/ParkingSpotRepository.java

```diff
@@ -2,7 +2,14 @@
 
 import com.exadel.sandbox.parking_spot.entity.ParkingSpot;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.time.LocalDate;
+import java.util.List;
 
 public interface ParkingSpotRepository extends JpaRepository<ParkingSpot, Long> {
     ParkingSpot findParkingSpotByOfficeId(Long id);
+
+    @Query(value = "select d.date from Booking b join b.parkingSpot p join b.dates d where p.id =:id")
+    List<LocalDate> findBookedDatesBySpotId(Long id);
```

Comment:
U use LocalDate cause we can't booked on some time, only for date?

Reply:
right. and I simply used date. for purpose then @evgeniy-milman suggested newer version of date which is compatible with sql.Date. So I used it